keyboards/formy46/keymaps/default/rules.mk

```diff
@@ -0,0 +1,11 @@
+# Formy46 Features Options
+SPLIT_KEYBOARD          =  yes       # Enable Split Keyboard
+RGB_MATRIX_ENABLE       =  yes       # Enable keyboard RGB Matrix
+RGB_MATRIX_DRIVER       =  WS2812    # RGB Matrix driver
+SRC                    +=  analog.c  # adc driver
+JOYSTICK_ENABLE         =  yes       # Enable Joystick
+POINTING_DEVICE_ENABLE  =  yes       # Enable Pointing Device
+ENCODER_ENABLE          =  yes       # Enable Rotary Encoder
+OLED_ENABLE             =  yes       # Enable OLED
+OLED_DRIVER             =  SSD1306   # OLED Type
+MOUSEKEY_ENABLE  		=  yes       # Enable Mouse keycord
```

Comment:
These should be defined at the keyboard level, and not the keymap level.

Reply:
I moved as much as I could to the keyboard.